### PR TITLE
Team5-Signoff-WorkSystem

### PR DESCRIPTION
**Pull request by jay-marcom**
requesting signoff for WorkSubsystem

### CODE REVIEW

**Review by JesseTNRoberts — COMMENTED** [review]
Comments given in person

**Comment by Baebel43** [comment]
This is out second attempt at a signoff for this system

File: Documentation/Signoffs/WorkSysten(1).md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+
+### Meeting the Ride Time Constraint
+
+Since there are many different trail lengths and skill levels, there is no set amount of duration for trails. Due to this the minimum amount of time that the batteries should last is hard to make a specific number for. Team 5 decided to set the minimum amount of time at 24 hours. This allows users to record large trails without having to worry about the batteries not lasting the entire trail. 
+
+- To solve for the maximum time the battery can last, the amperage output for each device and the milliamp hours output of the batteries is needed. The following values are needed: the current draw for the Zigbee Transmitter, the current draw for the Hall effect generator, and the current output of the battery. The values are 33 milliamps, 10 milliamps, and a minimum value for modern AA batteries is 2000 milliamp hours respectively. The following formulas can be used:
+
+**Amphours = Amps * Hour**
+
+**Hours = Amp Hours / Amps**
```
Comment (JesseTNRoberts) [inline Documentation/Signoffs/WorkSysten(1).md]: This is accurate only if the voltage required is equal for all devices and the battery. Is this the case? Also, AAs tend to drop voltage as they discharge. So, if this is a problem, the entire battery life won't be usable.

**Review by JesseTNRoberts — COMMENTED** [review]
The zigbee transceiver seems highly priced. Let's look into that and see if there is a cheaper option. I think that we can build without waterproofing for proof of concept. Leave the waterproofing items here in the design but exclude them from the PO.

File: Documentation/Signoffs/WorkSysten(1).md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+The Hall effect sensor has a quiescent voltage between 0.535 V and 0.665 V [1]. This is used for the range of the linear output voltage which has a minimum of the quiescent voltage and a maximum of the voltage delivered to the device (3.3 V or 5 V) minus 0.2 V [1]. Using an Arduino Nano 33 BLE we can set the voltage threshold to the driving voltage of the Hall effect sensor minus 0.2 V to be a 1 and anything below this value to be a 0.
+
+### Meeting the Diameter of Wheel Constraint
+
+The minimum wheel size has been set at 18 inches to account for the average wheel size of the smallest type of bike that would be commonly ridden on a trail, a BMX bike [3]. The largest wheel size has been set at 32 inches which is 3 inches above the most common largest mountain bike tire at 29 inches [4]. Additionally, this constraint for minimum wheel size is set to ensure that the device should not be fully submerged in water. If the rider were to hit a 9-inch deep puddle it would most likely not only result in the destruction of the device but also a failure of the bicycle.
+
+- This is proven within the calculations for velocity
+
+
+### Meeting the Sensing Speed Constraint
```
Comment (JesseTNRoberts) [inline Documentation/Signoffs/WorkSysten(1).md:52]: I don't understand how you are proving this. This needs to be more explicitly addressed. What is the nyquist rate for your application? How fast can you system sample? These have to be addressed clearly.

**Review by JesseTNRoberts — COMMENTED** [review]
Did not address previous feedback.

**Review by JesseTNRoberts — COMMENTED** [review]
devices don't have nyquist rates. Applications do. 

The calculations for the nyquist rate are not correct. Your analysis of this is not appropriate. Prove that this will detect the magnet each time it is under the sensor before the magnet leaves.

File: Documentation/Signoffs/WorkSysten(1).md (hunk 2 above)
Comment (JesseTNRoberts) [reply Documentation/Signoffs/WorkSysten(1).md:52]: Show the math and give comment where necessary. Don't talk about it in the abstract without actually doing the math.

**Review by JesseTNRoberts — COMMENTED** [review]
The analysis is very unclear. Use latex inline equations and show all the math with units. Add comments where necessary.

**Review by JesseTNRoberts — COMMENTED** [review]
You have proved that the sensor will respond in time. Good work! However, this doesn't prove that the overall system will be able to function. You haven't considered the Arduino and whether it can perform fast enough.

**Review by JesseTNRoberts — APPROVED** [review]

**Pull request merged**